plugins/base/src/test/kotlin/multiplatform/BasicMultiplatformTest.kt

```diff
@@ -25,6 +25,44 @@ class BasicMultiplatformTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    fun dataTestLongSparseArray() {
+        val testDataDir = getTestDataDir("multiplatform/longSparseArray").toAbsolutePath()
+
+        val configuration = dokkaConfiguration {
+            sourceSets {
+                sourceSet {
+                    name = "jvm"
+                    displayName = "JVM"
+                    sourceRoots = listOf("$testDataDir/jvmMain/")
+                }
+            }
+            sourceSets {
+                sourceSet {
+                    name = "common"
+                    displayName = "common"
+                    analysisPlatform = "common"
+                    sourceRoots = listOf("$testDataDir/commonMain/")
+                }
+            }
+            sourceSets {
+                sourceSet {
+                    name = "native"
+                    displayName = "Native"
+                    analysisPlatform = "native"
+                    sourceRoots = listOf("$testDataDir/nativeMain/")
+                }
+            }
+        }
+
+        testFromData(configuration) {
+            documentablesCreationStage = {
+                assertEquals(1, it[0].packages.size)
```

Comment:
It is not obvious. In creation stage modules come from PSI/Descriptors translators for each source set. We have two translators: `DefaultDescriptorToDocumentableTranslator` and `DefaultPSIToDocumentableTranslator`. In you case there are 3 source set. Then we will have 6 unmerged modules here:
`modules[0]` - java code of `jvm` source set.
`modules[1]` - kotlin code of `jvm` source set.
`modules[2]` - java code of `common` source set.
etc.

It is easier to use `documentablesMergingStage` instead of `documentablesCreationStage`.

Reply:
`documentablesMergingStage` and `documentablesTransformationStage` both only have jvm and native sourceSets